platform/error/platform-error-impl/package.json

```diff
@@ -44,7 +38,7 @@
     "node-fs": "0.1.7",
     "node-options": "0.0.3",
     "path": "0.4.9",
-    "casperjs": "~1.1.0",
+    "casperjs": "^1.1.0-beta3",
```

Comment:
This is actually the same exact casperjs  version we are already using (you can confirm in the "old" build log of one of these modules), the new npm is just more picky about how you specify this guy :/